bundles/org.openhab.binding.http/src/main/resources/OH-INF/update/instructions.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+<update:update-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:update="https://openhab.org/schemas/update-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/update-description/v1.0.0 https://openhab.org/schemas/update-description-1.0.0.xsd">
+
+	<thing-type uid="http:url">
+		<instruction-set targetVersion="1">
+			<add-channel id="lastSuccess">
+				<type>http:requestDateTime</type>
```

Comment:
```suggestion
				<type>http:request-datetime</type>
```

Reply:
Just out of curiosity, or those versions interchangeable, I would have thought you would need to recreate things when switching between them because of different namespaces?

Even with the SmartHome/J version existing and being used already, I would suggest to follow the naming convention of this project.

It will only require relinking of items when coming from the SmartHome/J version, but going forward we can add new channels according to the established cross-binding naming convention rather than being restricted to the old convention of the binding. I think that is a bigger advantage than the inconvenience of migrations.